apps/meteor/tests/end-to-end/api/livechat/11-livechat.ts

```diff
@@ -558,6 +558,22 @@ describe('LIVECHAT - Utils', () => {
 			const room = await createLivechatRoom(visitor.token);
 			await request.post(api('livechat/message')).set(credentials).send({ token: visitor.token, rid: room._id, msg: 'test' }).expect(200);
 		});
+		it("should set visitor's source as API after sending a message", async () => {
+			const visitor = await createVisitor();
```

Comment:
We need to delete it after all...

Reply:
Isn't it some sort of pattern from our side not to delete visitors and livechat rooms in tests? I recall @KevLehman mentioned something about this some time ago because that's an expensive behavior
None of the tests in this `describe` delete the visitor

Reply:
I think that's about rooms, If I recall correctly there's no even a way to delete it, only archive it. But we can remove visitors...